lib/links.js

```diff
@@ -42,6 +43,18 @@ class LinkParser {
     return result;
   }
 
+  getPRUrlsFromArray(arr) {
+    const result = [];
+    for (const item of arr) {
+      const m = item.match(PR_RE);
+      if (!m) continue;
+      const ref = m[1];
```

Comment:
Should this variable be named something along the lines of `prUrl` rather than `ref`?